.storybook/preview.js

```diff
@@ -1,8 +1,3 @@
-/*
-  * The addParameters and addDecorator APIs to add global decorators and parameters, exported by the various frameworks (e.g. @storybook/react) and @storybook/client were deprecated in 6.0 and have been removed in 7.0.
-
-Instead, use export const parameters = {}; and export const decorators = []; in your .storybook/preview.js. Addon authors similarly should use such an export in a preview entry file (see Preview entries).
-  * */
```

Comment:
We don't use `addParameters` or `addDecorator` and haven't been since the `v7` upgrade I think it's safe to remove these comments now